common/djangoapps/custom_settings/helpers.py

```diff
@@ -31,7 +31,8 @@ def validate_course_open_date(settings, course_open_date):
             course_open_date = course_open_date.replace(tzinfo=utc)
             course = CourseOverview.objects.get(id=settings.id)
 
-            if course.end < course_open_date or course_open_date < course.start:
+            if (course.end and course.end < course_open_date) or \
```

Comment:
I'm wondering why how this sanity was working in ficus. Would you please revert this change and verify again because I guess when you modified this sanity `CourseOverview` object was not being saved at the first place now we have fixed that, I believe it would work now.